src/components/fields/textfield/textfield.tsx

```diff
@@ -56,29 +56,24 @@ export const TextField = React.forwardRef<HTMLInputElement, IGenericFieldProps<I
 	// =============================================================================
 	// HELPER FUNCTIONS
 	// =============================================================================
-	const formatInputMode = () => {
-		let { inputMode } = "none" as TInputMode;
-
+	const formatInputMode = (): React.HTMLAttributes<HTMLInputElement>["inputMode"] => {
```

Comment:
note: `inputMode` from JSON schema will take precedence. otherwise we derive the inputMode based on the `type` defined in the JSON schema